File: C/process.c
#include <sys/types.h>
#include <stdio.h>
#include <unistd.h>

int main(){
	pid_t pid;	//Declarando variavel do tipo pid_t

	// Criando uma cópia do processo
	pid = fork();

	// Condição para caso de erro

	if(pid <0){
		fprintf(stderr,"Fork failed");
		exit(-1);//Saida forçada do sistema
	}
	
	// Condição para processo filho
	else if (pid ==0){
		execlp("/bin/ls","ls", NULL);
	}
	
	//Condição para processo pai
	else {
		//O pai só executa após a finalização do filho
		wait(NULL);
		printf("Child Complete");
		exit(0);
	}
}

File: Python/process.py
# Python code to create child process
import os from datetime
import datetime


# Criando uma cópia do processo
n = os.fork()

# Validação se o processo é p-filho ou p-pai (No caso é p-pai)

if n >0:
    print("Parent process id is :", os.getpid())

# Se n for menor que 0 é o p-filho (No caso é p-filho)

else: print("Child process id is :", os.getpid())

print(datetime.now())   #imprime a data atual
